polkadot/xcm/xcm-builder/src/origin_conversion.rs

```diff
@@ -332,3 +332,123 @@ impl<RuntimeOrigin: Clone, EnsureBodyOrigin: EnsureOrigin<RuntimeOrigin>, Body:
 		}
 	}
 }
+
+/// Implements `ConvertOrigin` to convert a `Location` into `RuntimeOrigin::root()`.
+///
+/// This is typically used when configuring `pallet-xcm` to allow a remote `Location`
+/// to act as the `Root` origin on the local chain.
+pub struct LocationAsSuperuser<SuperuserLocation, RuntimeOrigin>(
```

Comment:
please use it in some runtime configuration (e.g. Westend Relay) in this PR to show how it's meant to be used and also add an emulated test for it (if not already covered by some exiting ones)

Reply:
It is already used here: https://github.com/polkadot-fellows/runtimes/pull/626

We have also tests for all SP `fn governance_authorize_upgrade_works()` ready, but the plan was to actually use it on Donal's AHM branch directly, but we can also change XCM configurations on actual master, but I would better keep AHM related changes close to the Donal's AHM branch

Reply:
ok, sounds good